products/node-feature-discovery.md

```diff
@@ -25,8 +25,14 @@ identifiers:
 -   purl: pkg:github/kubernetes-sigs/node-feature-discovery
 -   purl: oci:node-feature-discovery?repository_url=registry.k8s.io/nfd
 
-# eol(x) = release(x+1)
+# eol(x) = releaseDate(x+2)
```

Comment:
how does this get determined?

Reply:
This is what's documented in the product description:

> Looking at prior releases, a new release is made every two weeks, and the latest two releases receive bug fixes.

Reply:
👍